website/docs/guides/dependencies.md

```diff
@@ -3,9 +3,11 @@
 Scarb manages dependencies by cloning packages from their Git repositories.
 To add a dependency, simply declare it in your `Scarb.toml`.
 
-> [!NOTE]
-> Using Git repositories as a foundation for package management is not an ideal
-> approach. Therefore, we plan to create a proper package registry in long term.
+> [!WARNING]
+> Using Git repositories as a foundation for package management is not the recommended approach anymore.
+> Instead, [registries](../registries/overview.md) are now the primary way to manage dependencies.
+> This guide will be updated to reflect the new approach soon.
+> For details on how to specify dependencies from the official registry, see [here](../reference/specifying-dependencies#specifying-dependencies-from-official-registry).
```

Comment:
I think we can address this in a separate issue, as rewriting / creating a separate guide will take some additional time, and I'm not sure this is necessary to close this PR and make docs available sooner.

However, if you think it's worth it to have this available right away, I can do it as a part of the same PR / stack.

Reply:
#1512

Reply:
IMHO this does not reflect the fact, that registry is still in beta.

Reply:
Included beta info in newest commit